src/cdk/a11y/BUILD.bazel

```diff
@@ -22,6 +17,17 @@ ng_module(
   tsconfig = ":tsconfig-build.json",
 )
 
+# TODO(jelbourn): remove this when sass_library acts like a filegroup
+filegroup(
```

Comment:
Now that we're in the business of making our own bazel rules, should we just replace these two rules with something like `material_sass_library`? or maybe go even further and extract all of the rules shared by every component into some sort of `material_module` rule?

```py
material_module(
  name = "button",
  module_name = "@angular/material/button",
  srcs = [...],
  deps = [...],
  scss = [...],
  theme_scss = [...],
  templates = [...]
)
```

Reply:
I'm planning on putting some time into either fixing or replacing the open source sass rules so that they match the ones in Google. These don't change often, so it shouldn't hurt our productivity to leave it until we can get the ecosystem sorted out.

Reply:
Generally we should try to make the primitives work instead of wrapping them in macros, which are leaky abstractions.